.github/workflows/integration_tests_deployment.yml

```diff
@@ -0,0 +1,68 @@
+name: Integration Tests Deployment
+on:
+  workflow_call:
+    inputs:
+      Environment:
+        type: string
+        required: true
+  workflow_dispatch:
+    inputs:
+      Environment:
+        type: choice
+        description: Choose Environment
+        default: dev
+        options:
+          - dev
+          - prd
+
+env:
+  AWS_REGION: ${{ secrets.AWS_REGION }}
```

Comment:
How come the region needs to be a secret instead of an environment variable?

Reply:
updated